charts/netbox/templates/worker/hpa.yaml

```diff
@@ -16,6 +16,43 @@ spec:
     name: {{ include "common.names.fullname" . }}-worker
   minReplicas: {{ .Values.worker.autoscaling.minReplicas }}
   maxReplicas: {{ .Values.worker.autoscaling.maxReplicas }}
+  {{- if .Values.worker.autoscaling.behavior }}
+  behavior:
+    {{- with .Values.worker.autoscaling.behavior.scaleUp }}
```

Comment:
Apologies if I was unclear, but I'd prefer to drop all `with` statements and use `if` everywhere possible.
Same in both files.